torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -155,6 +166,14 @@ struct ConstantPropagator {
     bool cond_val = constant_as<bool>(start_cond).value_or(true);
 
     bool loop_might_run = cond_val && iter_len > 0;
+    if (!loop_might_run)
+      GRAPH_UPDATE(
```

Comment:
Nit: use {} even if there is only one statement in the if.

Reply:
ok